### PR TITLE
fix: ELECTRON-1243: fix chrome flags tests

### PR DESCRIPTION
**Pull request by VishwasShashidhar**
## Description
Add support for disabling throttling and background rendering.

[ELECTRON-1243](https://perzoinc.atlassian.net/browse/ELECTRON-1243)

## Solution
Add a flag `disableThrottling` in `Symphony.config` that would allow users to disable throttling.

### CODE REVIEW

File: src/app/window-handler.ts [hunk 1]
```diff
@@ -89,9 +90,12 @@ export class WindowHandler {
         // Use these variables only on initial setup
         this.config = config.getConfigFields([ 'isCustomTitleBar', 'mainWinPos', 'minimizeOnClose', 'notificationSettings' ]);
         this.globalConfig = config.getGlobalConfigFields([ 'url', 'contextIsolation' ]);
+        const { customFlags } = config.getGlobalConfigFields([ 'customFlags' ]) as IConfig;
```
Comment (KiranNiranjan) [inline src/app/window-handler.ts]: This can be merged with the above line

File: src/app/window-handler.ts [hunk 2]
```diff
@@ -89,9 +90,12 @@ export class WindowHandler {
...
+        this.globalConfig.customFlags = customFlags;
 
         this.windows = {};
         this.contextIsolation = this.globalConfig.contextIsolation || false;
+        this.backgroundThrottling = !this.globalConfig.customFlags.disableThrottling;
```
Comment (KiranNiranjan) [inline src/app/window-handler.ts]: we can read `disableThrottling` directly from `customFlags`

File: src/app/window-handler.ts (hunk 1 above)
Comment (VishwasShashidhar) [reply src/app/window-handler.ts]: Addressed

File: src/app/window-handler.ts (hunk 2 above)
Comment (VishwasShashidhar) [reply src/app/window-handler.ts]: Addressed as part of the above comment.

**Pull request merged**